### PR TITLE
re:TerraGov

### PR DESCRIPTION
**Pull request by triplezeta**
## About The Pull Request

Okay, before I pass out for the night (morning??) I wanted to get this done. Basically, this replaces most instances of the phrase "TerraGov" with the phrase "SolGov" along with "terra" to "sol" and such. 
This also gives SolGov loadouts the M9C by default in their pocket along with extra magazines, moves the SolGov AR to the belt slot on the elite, and replaces the emags with a couple sticks of X4.

To do: 

- [x] replace shitty critawakets sprites 
-  [x] ~~add the MP5C, SolGov's SMG for spray-and-pray combat~~
- [x] ~~throw the terragov laser and pulse guns in the trash and add the SolGov Laser Cannon~~
- [x] ~~do a balance pass of SolGov's stuff~~
Maldo Edit: The above stuff can happen in another PR, lets pick up these changes and reduce our number of active test merges.

## Why It's Good For The Game

SolGov has potential to be something great. All we have to do is throw out the crappy sprites and maintain it.

## Changelog
:cl:
tweak: most instances of "TerraGov" on equipment have now been appropriately tagged as "SolGov" equipment
balance: SolGov loadouts have X4 instead of emags
imageadd: new SolGov sprites (when i get around to it)
imagedel: old TerraGov sprites (when i get around to it)
/:cl:

### CODE REVIEW

**Comment by Superyodeler** [comment]
7

**Comment by Superyodeler** [comment]
If you've finished stuff on the to-do list, make sure you check it off so I can see.

**Comment by triplezeta** [comment]
i have not

**Comment by Maldaris** [comment]
Is there any reason we can't just take the changes we currently have and merge them, and make new PRs as the other work becomes completed on this?

**Comment by triplezeta** [comment]
;


no

**Comment by triplezeta** [comment]
actually wait maybe

**Comment by triplezeta** [comment]
ok no

**Comment by Maldaris** [comment]
> ok no

Why?

**Comment by triplezeta** [comment]
i mean like

no for "is there something stopping us from merging this right now?"

you could merge it right now if you wanted to

**Comment by Maldaris** [comment]
> i mean like
> 
> no for "is there something stopping us from merging this right now?"
> 
> you could merge it right now if you wanted to

It's your PR ultimately.

**Comment by triplezeta** [comment]
yeah I don't have any problems with this being merged right now

File: waspstation/code/modules/clothing/head/berets.dm [hunk 1]
```diff
@@ -211,3 +211,29 @@
...
+	armor = list("melee" = 40, "bullet" = 30, "laser" = 30, "energy" = 40, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 60)
+	strip_delay = 60
+
+/obj/item/clothing/head/beret/solgov/plain
+	name = "\improper SolGov beret"
+	desc = "A plain blue beret. It looks like it's missing something."
+	icon_state = "beret_solgovplain"
+	item_state = "beret_solgovplain"
+
+/obj/item/clothing/head/beret/solgov/terragov
```
Comment (Maldaris) [inline waspstation/code/modules/clothing/head/berets.dm:229]: If we're dropping the terragov name, is there a reason we're keeping these around?

File: waspstation/code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -1,10 +1,26 @@
-/obj/item/clothing/head/helmet/terra
-	name = "terragov helmet"
-	desc = "A helmet manufactured by TerraGov to protect craniums."
-	icon_state = "terrahelmet"
-	item_state = "terrahelmet"
+/obj/item/clothing/head/helmet/solgov
+	name = "\improper SolGov Helmet"
+	desc = "A helmet manufactured by SolGov to protect craniums. This one is painted in original TerraGov green."
```
Comment (Maldaris) [inline waspstation/code/modules/clothing/head/helmet.dm:3]: Same as above? Maybe I'm unclear as to what's going on with the name change.

File: waspstation/code/modules/clothing/head/berets.dm (hunk 1 above)
Comment (triplezeta) [reply waspstation/code/modules/clothing/head/berets.dm:229]: they're Old terragov clothes back when Solgov was still Terragov

File: waspstation/code/modules/clothing/head/helmet.dm (hunk 2 above)
Comment (triplezeta) [reply waspstation/code/modules/clothing/head/helmet.dm:3]: ok now that I think of it I may still have things to do

**Review by Superyodeler — APPROVED** [review]

**Review by Maldaris — APPROVED** [review]

**Pull request merged**